SCAN.md

```diff
@@ -1,5 +1,24 @@
 Documentation on the Seattle Coronavirus Assessment Network (SCAN) project
 
+- [Adding a new study arm](#adding-a-new-study-arm)
+- [Adding a new language](#adding-a-new-language)
+
+## Adding a new study arm
+
+1. First, answer the questions listed in the [barcodes documentation] pertaining to creating new collections.
+2. Create a new identifier set.
+3. Create a new barcode label layout for the [labelmaker].
+4. Enable REDCap DETs for the new project once testing is complete and it's ready for production.
```

Comment:
I couldn't figure out where to do this in RedCAP. Let's add this to the step:
Within the REDCap project, go to Project Setup, find and click the Additional Customizations button, scroll to the bottom of the modal, and find the Data Entry Trigger section.

Reply:
That make sense. I almost forget where that button is every time. This task appears in a couple of places, so this makes me inclined to create a REDCap documentation page where we describe tasks like this. I'll push a fix.

Reply:
I added your steps to a [new wiki page](https://github.com/seattleflu/documentation/commit/e65c5a8d09afeefc9cd0e11ba808e7d22039ad2a).